package.json

```diff
@@ -105,7 +105,7 @@
     "moment": "2.22.1",
     "react-beautiful-dnd": "2.6.2",
     "react-color": "2.13.1",
-    "react-datepicker": "1.3.0",
+    "react-datepicker": "1.7.0",
```

Comment:
Are you sure about updating this package version?

Reply:
Updating the package version fixes this issue: https://docs.google.com/document/d/10Sc94fHOQiUOYGkDIaj0eDIpF5FawXrj_qk44g6salQ/edit#bookmark=id.x7ruash6ye37

Reply:
1. Did you check that everything works as before?
2. Did you check the other components that use the TimePicker component?